lib/byron/test/unit/Cardano/Wallet/Byron/NetworkSpec.hs

```diff
@@ -41,20 +45,23 @@ spec = describe "getTxParameters" $ do
     it "Correct values are queried" $ do
         withTestNode $ \gbp sock vData -> withLogging $ \(tr, getLogs) -> do
             -- Initial TxParameters for NetworkLayer are all zero
-            let gbp' = gbp { txParameters = zeroTxParameters }
+            let pps = protocolParameters gbp
+            let pps' = pps { txParameters = zeroTxParameters }
+            let gbp' = gbp { protocolParameters = pps' }
```

Comment:
A lens operator might look better here.

Reply:
Fixed in 0fc13db89eec38e41f21166d3555197a435699c2.